src/main/scala/nl/vroste/zio/kinesis/client/DynamicConsumer.scala

```diff
@@ -40,7 +40,8 @@ object DynamicConsumer {
    * @param cloudWatchClientBuilder
    * @param dynamoDbClientBuilder
    * @param isEnhancedFanOut Flag for setting retrieval config - defaults to `true`. If `false` polling config is set.
-   * @tparam R
+   * @param maybeTableName Optionally set the lease table name - defaults to None. If not specified the `applicationName` will be used.
```

Comment:
The 'maybe' is kind of implied by the `Option`, don't you think? How about `leaseTableName`?

Reply:
:+1: